app/scripts/lib/experiments/grouping-rules/base.js

```diff
@@ -65,4 +65,14 @@ module.exports = class BaseGroupingRule {
     }
     throw new Error('choose must be overridden');
   }
+
+  /**
+   * Is this a test email?
+   *
+   * @param {String} email
+   * @returns {Boolean}
+   */
+  isTestEmail (email) {
+    return /(.+@mozilla\.(com|org)$)|(.+@softvision\.(com|ro)$)/.test(email);
```

Comment:
No biggie, but including the `$` separately in each arm of the outer `|` suprised me slightly. If you're matching the end of the string, why not just match it once at the end of the expression?